src/externalized/DefaultContentManager.cc

```diff
@@ -898,6 +898,7 @@ void DefaultContentManager::loadStringRes(const ST::string& name, std::vector<co
 	case GameVersion::POLISH:       fullName += "-pl";    break;
 	case GameVersion::RUSSIAN:
 	case GameVersion::RUSSIAN_GOLD: fullName += "-rus";   break;
+    case GameVersion::CHINESE: fullName += "-chs";   break;
```

Comment:
tabs for indentation

Reply:
Thanks for replay.
Which indent tool and config should I use in this project.

Reply:
For rust just use rustfmt, while for the inherited c++ code, we don't have a formalized standard yet. There was some discussion in a still open issue, but nothing has been codified. Standardizing indentation was a big first step a few years back, since we got the code in a mixed state.

Rule of thumb: match the indentation style in the surrounding code.

Reply:
Some guidance here: https://github.com/ja2-stracciatella/ja2-stracciatella/blob/master/CONTRIBUTING.md#axioms-of-style

Reply:
Thanks, I will fix them.